doc/languages.rst

```diff
@@ -1252,12 +1252,20 @@ to view the docstring of the syntax checker.  Likewise, you may use
 
       Check syntax with `XMLStarlet <http://xmlstar.sourceforge.net>`_.
 
+      .. defcustom:: flycheck-xml-xmlstarlet-xsd-path
```

Comment:
For the documentation, I would rather merge the two options in one `defcustom` block, since they are the same for different checkers.  Look at how `flycheck-clang-args` and `flycheck-gcc-args` are defined in the doc.